test-integ/topoutil/fixtures.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package topoutil
+
+import (
+	"strconv"
+
+	"github.com/hashicorp/consul/testing/deployer/topology"
+)
+
+const HashicorpDockerProxy = "docker.mirror.hashicorp.services"
+
+func NewFortioServiceWithDefaults(
+	cluster string,
+	sid topology.ServiceID,
+	nodeVersion topology.NodeVersion,
```

Comment:
This was moved mostly the same save for the `nodeVersion` param and behavior below.